Makefile

```diff
@@ -199,3 +203,13 @@ $(LICHE):
 
 $(PROMU):
 	$(call fetch_go_bin_version,github.com/prometheus/promu,$(PROMU_VERSION))
+
+$(PROTOC):
+	@echo ">> fetching protoc@${PROTOC_VERSION}"
+	@if [ ! -x '$(BIN_DIR)/protoc' ]; then \
+		curl -LSs $(PROTOC_DOWNLOAD_URL) -o $(BIN_DIR)/$(PROTOC_PACKAGE); \
```

Comment:
I messed up... tmp packages in bin might be not ok... ): Can we move it back to tmp. 
What I meant with my above comments is for `$(PROTOC)` itself - it should be in `BIN_DIR` as you already did it.

Sorry for confusion!